geo_to_hca/geo_to_hca.py

```diff
@@ -0,0 +1,301 @@
+# --- core imports
+import argparse
+import os
+from pathlib import Path
+import sys
+
+# --- third-party imports
+import pandas as pd
+from openpyxl import load_workbook, Workbook
+
+# --- application imports
+from geo_to_hca.utils import get_tab
+from geo_to_hca.utils import parse_reads
+from geo_to_hca.utils import sra_utils
+from geo_to_hca.utils import utils
+
+DEFAULT_HCA_TEMPLATE = Path(__file__).resolve().parents[1] / "template/hca_template.xlsx"
+"""
+Define functions.
+"""
+def fetch_srp_accession(geo_accession: str) -> str:
+    """
+    Function to retrieve an SRA study accession given a GEO accession.
+    """
+    srp = sra_utils.SraUtils.get_srp_accession_from_geo(geo_accession)
+    if not srp.empty:
+        if srp.shape[0] == 1:
+            srp = srp.iloc[0]["study_accession"]
+        elif srp.shape[0] > 1:
+            print("More than 1 accession has been found. Please enter re-try with a single SRA Study accession.")
+            sys.exit()
+    else:
+        print("Could not recognise GEO accession %s; is it a GEO Superseries? If yes, please re-try with a subseries accession: " % (geo_accession))
```

Comment:
Please use logging here. Or at least, when you use it in the broker, make sure stdout goes through the logger.

Reply:
These print statements did go through the [broker logging, which was to stout ](https://github.com/ebi-ait/ingest-broker/blob/fc2275cbefd6c03f60cf27496c7cfff1c25cfee6/broker_app.py#L31).

Replaced the print statements with logging